### PR TITLE
connector inoperative overridden

### PR DESCRIPTION
**Pull request by james-ctc**
## Describe your changes

When OCPP ChangeConfiguration to Inoperative is set this is overridden on EVerest restart.
In addition some actions on the CP line were causing the state to leave DISABLED.

This work is based upon:
https://github.com/EVerest/everest-core/pull/505

This changed is linked with 
https://github.com/EVerest/libocpp/pull/623

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [X] I read the [contribution documentation](https://github.com/EVerest/EVerest/blob/main/CONTRIBUTING.md) and made sure that my changes meet its requirements

### CODE REVIEW

File: types/evse_manager.yaml [hunk 1]
```diff
@@ -553,4 +591,4 @@ types:
           type: object
           $ref: /evse_manager#/Connector
         minItems: 1
-        maxItems: 128
+        maxItems: 128
```
Comment (barsnick) [inline types/evse_manager.yaml:594]: No newline at end of file

File: types/evse_manager.yaml [hunk 2]
```diff
@@ -324,6 +324,39 @@ types:
...
+        enum:
+        - Unspecified
+        - LocalAPI
+        - LocalKeyLock
+        - ServiceTechnician
+        - RemoteKeyLock
+        - MobileApp
+        - FirmwareUpdate
+        - CSMS
+      enable_state:
```
Comment (barsnick) [inline types/evse_manager.yaml:348]: Trailing whitespace

File: types/evse_manager.yaml (hunk 1 above)
Comment (james-ctc) [reply types/evse_manager.yaml:594]: more applicable to #505 where it will need to be addressed.

File: types/evse_manager.yaml (hunk 2 above)
Comment (james-ctc) [reply types/evse_manager.yaml:348]: more applicable to #505 where it will need to be addressed.

**Comment by Pietfried** [comment]
This issue has been adressed by: https://github.com/EVerest/everest-core/pull/714

**Pull request closed without merging**